### ISSUE TITLE
Add support for cache statistics

### ISSUE THREAD

**Issue by ppanero** [rtreffer/minidns#57]
Add support for cache statistics. I offer myself to implement this feature, but I would like to have a chat in order to discuss possible implementations (Gitter?)

**Comment by Flowdalic** [comment]
LRUCache has already some support for statistics. See e.g. https://github.com/rtreffer/minidns/blob/563774eba002d69ed2141487ab2a05444d81645b/minidns-core/src/main/java/de/measite/minidns/cache/LRUCache.java#L154

**Comment by ppanero** [comment]
Yeah the attributes are there. I meant more like adding a few functions which return e.g. usage rate, miss rate, hit rate, etc. (And separating them on positive/negative caching). So no adding it, but "improving" it
